setup.cfg

```diff
@@ -48,3 +48,7 @@ disallow_any_generics = True
 check_untyped_defs = True
 
 ignore_missing_imports = True
+
+
+[aliases]
```

Comment:
Could we put this at the top, so as to keep the alphabetical sorting of sections?

Reply:
6df539e